src/Unread.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
             return false;
         } else if (ev.getType() == 'm.call.answer' || ev.getType() == 'm.call.hangup') {
             return false;
-        } else if (ev.getType == 'm.room.message' && ev.getContent().msgtype == 'm.notify') {
+        } else if (ev.getType() == 'm.room.message' && ev.getContent().msgtype == 'm.notify') {
```

Comment:
I'm not entirely sure this is a thing anymore - it is probably safe to drop `m.notify` as a special case. It doesn't appear anywhere in the spec, synapse, or current riot stack, nor in any relevant issues. Matthew or @dbkr might know more about its history though - last reference I can find is 2015.

Reply:
Dave says it's still a thing. Bots apparently use it.